src/test_snmp.cpp

```diff
@@ -86,6 +88,190 @@ TEST_F(SNMPTest, LatencyCalculations)
   delete tbl;
 }
 
+TEST_F(SNMPTest, LatencyScopeCalculations)
+{
+  cwtest_completely_control_time(true);
+
+  // Create a table.  Use a different OID base because otherwise the indices
+  // clash with later tests.
+  SNMP::EventAccumulatorByScopeTable* tbl = SNMP::EventAccumulatorByScopeTable::create("latency", test_oid);
```

Comment:
Just seen the commit comment saying no need to use different OIDs. Can you update the comments to reflect this?